index.html

```diff
@@ -11,9 +11,197 @@
+                        <td data-label="State">California</td>
+                        <td data-label="Phone">+1764156161</td>
+                        <td data-label="Email">chandersen@pixel.com </td>
+                    </tr>
+                    <tr>
+                        <td data-label="Navn">Christian Andersen</td>
+                        <td data-label="Company">Pixel Inc.</td>
+                        <td data-label="Level">Senior</td>
+                        <td data-label="Deals closed">315</td>
+                        <td data-label="State">California</td>
+                        <td data-label="Phone">+1764156161</td>
+                        <td data-label="Email">chandersen@pixel.com </td>
+                    </tr>
+                    <tr>
+                        <td data-label="Navn">Christian Andersen</td>
+                        <td data-label="Company">Pixel Inc.</td>
+                        <td data-label="Level">Senior</td>
+                        <td data-label="Deals closed">315</td>
+                        <td data-label="State">California</td>
+                        <td data-label="Phone">+1764156161</td>
+                        <td data-label="Email">chandersen@pixel.com </td>
+                    </tr>
+                    <tr>
+                        <td data-label="Navn">Christian Andersen</td>
+                        <td data-label="Company">Pixel Inc.</td>
+                        <td data-label="Level">Senior</td>
+                        <td data-label="Deals closed">315</td>
+                        <td data-label="State">California</td>
+                        <td data-label="Phone">+1764156161</td>
+                        <td data-label="Email">chandersen@pixel.com </td>
+                    </tr>
+                    <tr>
+                        <td data-label="Navn">Christian Andersen</td>
+                        <td data-label="Company">Pixel Inc.</td>
+                        <td data-label="Level">Senior</td>
+                        <td data-label="Deals closed">315</td>
+                        <td data-label="State">California</td>
+                        <td data-label="Phone">+1764156161</td>
+                        <td data-label="Email">chandersen@pixel.com </td>
+                    </tr>
+                    <tr>
+                        <td data-label="Navn">Christian Andersen</td>
+                        <td data-label="Company">Pixel Inc.</td>
+                        <td data-label="Level">Senior</td>
+                        <td data-label="Deals closed">315</td>
+                        <td data-label="State">California</td>
+                        <td data-label="Phone">+1764156161</td>
+                        <td data-label="Email">chandersen@pixel.com </td>
+                    </tr>
+                </tbody>
+                <tfoot>
+                    <tr>
+                        <td colspan="4">Total agents: 28 </td>
```

Comment:
It's recommended to have one td inside tfoot. In this case, there is no need to track if the content of each td (those 3 that there are) is longer than td itself. Using an approach with one td and assigning styles (flex) for inner elements works best.